parsers/julia.c

```diff
@@ -846,6 +886,29 @@ static void addTag (vString* ident, const char* type, const char* arg_list, int
     makeTagEntry(&tag);
 }
 
+static void addReferenceTag (vString* ident, int kind, int role, unsigned long line, MIOPos pos, vString* scope)
+{
+    if (kind == K_NONE)
+    {
+        return;
+    }
+    tagEntryInfo tag;
+    // HELP WANTED: for some reason initRefTagEntry doesn't work
+    // initRefTagEntry(&tag, vStringValue(ident), kind, role);
+    initTagEntry(&tag, vStringValue(ident), kind);
+    tag.lineNumber = line;
+    tag.filePosition = pos;
+    tag.sourceFileName = getInputFileName();
+    tag.extensionFields.signature = NULL;
+    //tag.extensionFields.roleBits = makeRoleBit(role);
```

Comment:
You may not need this line.
 initRefTagEntry may set the bit.

Reply:
Done.